alternator/executor.cc

```diff
@@ -98,7 +102,7 @@ std::string make_jsonable::to_json() const {
 }
 
 json::json_return_type make_streamed(rjson::value&& value) {
-    // CMH. json::json_return_type uses std::function, not noncopyable_function. 
+    // CMH. json::json_return_type uses std::function, not noncopyable_function.
```

Comment:
Please don't improve unrelated whitespaces in your patch. You can make it a separate patch in the same PR if it's important to you.

Reply:
That's my IDE insists of fixing white space, it's by mistake that I pushed it